apps/total-typescript/src/templates/exercise-template.tsx

```diff
@@ -50,9 +51,7 @@ const ExerciseTemplate: React.FC<{
           {lessonSlug: router.query.lesson as string},
           {
             onSettled: () => {
-              utils.moduleProgress.bySlug.invalidate({
-                slug: module.slug.current,
-              })
+              client.invalidateQueries()
```

Comment:
invalidation via the query client functions as expected and re-renders the with updated progress across the page